twint/url.py

```diff
@@ -110,7 +110,7 @@ async def Search(config, init):
         q += f" geocode:{config.Geo}"
     if config.Search:
 
-        q += f" {config.Search}"
+        q += f" {config.Search} lang:{config.Lang}"
```

Comment:
Hi,
thanks for putting up the fix for this issue.
Although this fix would not work if config.Lang is not explicitly specified while creating the config object. also if you don't specify the --lang = "xyz" in the command line (using twint from command line), it'll again break.

So, instead of adding `lang:{config.Lang}` after the search query (Line 113)
simply replace Lines 102 & 103 with `q += f" lang:{config.Lang}"` and it should fix the issue for good.

Reply:
Tried as himanshudabas's solution but it didn't work again. For cmd, --lang "tr" didn't worked

Reply:
Can you try to follow the steps mentioned in #1053 ?